docs/reference/esql/processing-commands/where.asciidoc

```diff
@@ -58,6 +58,23 @@ For a complete list of all functions, refer to <<esql-functions>>.
 
 include::../functions/predicates.asciidoc[tag=body]
 
+For matching text, you can use <<esql-search-functions,full text search functions>> like `MATCH`.
+
+Use <<esql-match,`MATCH`>> to perform a <<query-dsl-match-query,match query>> on a specified field.
+
+Match can be used on text fields, as well as other field types like boolean, dates, and numeric types.
+
+[source.merge.styled,esql]
+----
+include::{esql-specs}/match-function.csv-spec[tag=match-with-field]
+----
+[%header.monospaced.styled,format=dsv,separator=|]
+|===
+include::{esql-specs}/match-function.csv-spec[tag=match-with-field-result]
+|===
+
+For a simplified syntax, you can use the <<esql-search-operators,match operator>> `:` instead of `MATCH`.
```

Comment:
```suggestion
[TIP]
====
You can also use the shorthand <<esql-search-operators,match operator>> `:` instead of `MATCH`.
====
```

Reply:
This examples section is getting dense and we don't have subheadings at the moment so this little tip callout will provide a bit of visual differentiation :)

Minor optional rewording for brevity

Reply:
Nice! Thanks @leemthompo